content/sensu-go/6.0/reference/entities.md

```diff
@@ -66,6 +66,32 @@ You can modify the proxy entity later if needed.
 
 Use [proxy entity filters][19] to establish a many-to-many relationship between agent entities and proxy entities if you want even more power over the grouping.
 
+## Create and modify agent entities
+
+When an agent connects to a backend, the agent entity defnition is created from the information in `/etc/sensu/agent.yml`.
+
+You can modify agent entities with [sensuctl][32], the [entities API][31], and the [web UI][33].
+
+{{% notice note %}}
+**NOTE**: You cannot modify agent entities with `/etc/sensu/agent.yml`.
+The entity attributes in `/etc/sensu/agent.yml` are used only for initial entity creation.
```

Comment:
Same thing here, `/etc/sensu/agent.yml` is only the _default_ config file path for linux.